twindle-cli/twitter/utils/twitter-api-keys.js

```diff
@@ -1,2 +1,12 @@
-const BEARER_TOKEN = "Bearer " + process.env.TWITTER_AUTH_TOKEN;
+const { UserError } = require("../../helpers/error");
+
+const twitterAuthToken = process.env.TWITTER_AUTH_TOKEN;
+
+if (!twitterAuthToken)
+  throw new UserError(
+    "bearer-token-not-provided",
+    "Please ensure that you have a .env file containing a value for TWITTER_AUTH_TOKEN"
+  );
```

Comment:
@PuruVJ I think it would make sense to have specific functions exported for common errors so that we won't duplicate the error code and message. But in this case, I suppose it is fine for now.

Reply:
Yes @PuruVJ has implemented some user defined errors for now. But we need to come up with possible error scenarios and how best to handle them:
* Network connectivity issues in which case fetch throws an error
* No or invalid access token from Twitter in which case Twitter throws a 401 response
* If tweet is older than 7 days dont process because then we will get an incomplete set of tweets for the thread
* Tweet id is for a deleted tweet
* If one of the errors above happens or if no tweets are found - dont create the PDF and dont print the your tweets are saved to pdf message

Maybe there are more errors- we might need to track them